oracle-plugin/src/e2e-test/java/io.cdap.plugin/oracle/runners/TestRunner.java

```diff
@@ -26,7 +26,9 @@
 @CucumberOptions(
   features = {"src/e2e-test/features"},
   glue = {"stepsdesign", "io.cdap.plugin.common.stepsdesign", "io.cdap.plugin.oracle.stepsdesign"},
-  tags = {"@Oracle"},
+  tags = {"@Oracle and not @PLUGIN-1535"},
```

Comment:
Why do we need this change ?

Reply:
In one of the Oracle to BQ scenario of the runtime feature, which uses the existing datatypes hook, the pipeline fails with issue as mentioned in the raised bug(PLUGIN-1535). Hence tag was added so that the scenario runs after the fix and change made in runner file. Later after the fix the tag can be removed from runner file.

Reply:
We can remove this tag now. The PLUGIN-1535 is now resolved and closed.

Reply:
Removed the tag.